controllers/transactionsController.js

```diff
@@ -0,0 +1,39 @@
+const transactionsModel = require('../models/transaction')
+const { getUserId } = require('../utils/users')
+
+/**
+ * Collects all transactions and sends only required data for username spicified in url
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const fetchLatest = async (req, res) => {
+  try {
+    const userId = await getUserId(req.params.username)
+    if (userId !== undefined) {
+      const noOfRecords = req.query.noOfRecords
```

Comment:
Does the query param need to be this long? Can it be simply `n`? We could specify what `n` is in API contract

Also, wouldn't it be better if the default value `10` is set here itself (using `||` operator)

Reply:
Order = o (ASC or DESC)
NoOfRecords = n (1,2,3....)

Reply:
Done